mTransit/src/providers/google-maps.ts

```diff
@@ -297,14 +297,13 @@ export class GoogleMaps {
 
     marker.addListener('click', () => {
     marker.setIcon(selectedMarker);
-    infowindow.open(this.map, marker);
+    //infowindow.open(this.map, marker);
```

Comment:
In this example, I commented out infoWindow incase we ever use it again to "confirm" our users